src/main/java/com/algorand/algosdk/v2/client/common/AlgodClient.java

```diff
@@ -140,12 +142,12 @@ public GetBlock GetBlock(Long round) {
     }
 
     /**
-     * Get a Merkle proof for a transaction in a block.
+     * Get a proof for a transaction in a block.
      * /v2/blocks/{round}/transactions/{txid}/proof
      */
-    public GetProof GetProof(Long round,
+    public GetTransactionProof GetTransactionProof(Long round,
```

Comment:
The API now supports one more parameter which is the hash type
should we add a parameter to this function?

Reply:
That is an optional query parameter, the arguments here only include required path parameters.

Reply:
You are right. it is indeed an optional param. However, this API is a  significant part of "how to build a light client" and how to bridge tokens, and currently, its default value does not match the light client verification so it would make it usable for the light client use case

Reply:
It's  _not_ unusable--required parameters go in the initial constructor, and optional parameters can be specified in a builder-like pattern like so
```java
TransactionProofResponse resp = client.GetTransactionProof(123456, "abc123").hashtype(Enums.HashType("sha256_512")).execute();
```